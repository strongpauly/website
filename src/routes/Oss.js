import React, { Component } from 'react';

export default class Oss extends Component {

  render() {
    return <div id="ossContainer" className="container">
      <p>My personal github account can be found <a href="https://www.github.com/strongpauly">here</a>.</p>
      <p>All of the source for client based projects demoed on this site can
      be seen linked to that account as well as some library or middle tier projects.  These include: </p>
      <ul>
        <li><a href="https://www.github.com/strongpauly/betterset">BetterSet</a>.
          An extension of the ES6 Set object which includes set operation methods.</li>
      </ul>
    </div>;
  }

}
