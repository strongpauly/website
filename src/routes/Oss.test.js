import React from 'react';
import {Oss} from './Oss';
import {shallow} from 'enzyme';

/* eslint-env jest */

describe('Home', () => {
  it('renders without crashing', () => {
    const app = shallow(<Oss/>);
    expect(app).toMatchSnapshot();
  });

  it('creates a #homeContainer', () => {
    const app = shallow(<Oss/>);
    expect(app.find('#ossContainer')).toHaveLength(1);
  });
});
