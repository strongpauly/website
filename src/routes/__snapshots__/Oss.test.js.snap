// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders without crashing 1`] = `
<div
  id="ossContainer"
>
  <p>
    My personal github account can be found 
    <a
      href="https://www.github.com/strongpauly"
    >
      here
    </a>
    .
  </p>
  <p>
    All of the source for client based projects demoed on this site can be seen linked to that account as well as some library or middle tier projects.  These include: 
  </p>
  <ul>
    <li>
      <a
        href="https://www.github.com/strongpauly/betterset"
      >
        BetterSet
      </a>
      . An extension of the ES6 Set object which includes set operation methods.
    </li>
  </ul>
</div>
`;
